Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/Feature/NodeMove.php

```diff
@@ -41,6 +42,9 @@ abstract protected function getTableNamePrefix(): string;
     private function whenNodeAggregateWasMoved(NodeAggregateWasMoved $event): void
     {
         $this->transactional(function () use ($event) {
+
+            $dimensionSpacePoints = [];
+
```

Comment:
why is this needed? you never read from it

Reply:
probably a left-over, I'll check